packages/genesys-spark-components/.stylelintrc.json

```diff
@@ -7,7 +7,7 @@
         "src/components/stable/gux-button/**/*.{scss,html}",
         "src/components/stable/gux-button-slot/**/*.{scss,html}",
         "src/components/beta/**/*.{scss,html}",
-        "src/components/stable/gux-[a-m]**/*.scss"
+        "src/components/stable/gux-**/*.scss"
```

Comment:
Leaving the `gux-` part in for now because I'm kicking nested components to another PR